src/apps/search-result/search-result.tsx

```diff
@@ -98,6 +99,24 @@ const SearchResult: React.FC<SearchResultProps> = ({ q, pageSize }) => {
     }
   }, [campaignFacets, mutate]);
 
+  // Check for material type filters in url on pageload
+  // (this part will be reworked later based on a change request from the client)
```

Comment:
I do not think "(this part will be reworked later based on a change request from the client)" is very helpful apart from signaling that we expect this to change.

1. It refers to process "change request" and "client" which will mean little to the average reader.
2. It does not explain *why* we expect this to be reworked

I suggest

```suggestion
  // This is an initial, intentionally simple approach supporting what is required by the search header.
  // It could be reworked to support all filters and terms at a later point.
```